apps/arweave/src/ar_coordination.erl

```diff
@@ -2,15 +2,16 @@
 
 -behavior(gen_server).
 
--export([start_link/0, get_state/0]).
+-export([start_link/0, get_state/0, is_peer/1]).
 
 -export([init/1, handle_cast/2, handle_call/3, handle_info/2, terminate/2]).
 
 -include_lib("arweave/include/ar.hrl").
 -include_lib("arweave/include/ar_config.hrl").
 
 -record(state, {
-    mining_peers = #{}
+    mining_peers = #{},
+	peers = []
```

Comment:
If we are having two views of almost the same data we should name them accordingly. Something like `peers_by_partition` and `peer_list` to let the reader know what's each one.